src/components/Kv/KvLightbox.vue

```diff
@@ -1,25 +1,26 @@
 <template>
-	<focus-lock :disabled="!isShown">
-		<transition name="kvfade">
-			<div
-				v-show="isShown"
-				class="kv-lightbox"
-				:class="{
-					'kv-lightbox--inverted': inverted,
-					'kv-lightbox--no-padding-top': noPaddingTop,
-					'kv-lightbox--no-padding-bottom': noPaddingBottom,
-					'kv-lightbox--no-padding-sides': noPaddingSides,
-					'kv-lightbox--full-width': fullWidth,
-				}"
-				data-test="kv-lightbox"
-				ref="kvlightbox"
-				@click.stop.prevent="closeLightbox"
-				role="dialog"
-				:aria-labelledby="title ? 'lightbox-title' : null"
-			>
+	<transition name="kvfade">
+		<div
+			class="kv-lightbox"
+			:class="{
+				'kv-lightbox--inverted': inverted,
+				'kv-lightbox--no-padding-top': noPaddingTop,
+				'kv-lightbox--no-padding-bottom': noPaddingBottom,
+				'kv-lightbox--no-padding-sides': noPaddingSides,
+				'kv-lightbox--full-width': fullWidth,
+			}"
+			data-test="kv-lightbox"
+			@click.stop.prevent="closeLightbox"
+			v-show="isShown"
+		>
+			<focus-lock v-if="isShown" :return-focus="true">
```

Comment:
the `return-focus` prop fixes a bug we have in prod where the focus doesn't return to the original button when the dialog closes

Reply:
Also had to change from :disabled to v-if in order to get the return focus piece working. (The element needs to leave the DOM for it to work)